data_server.py

```diff
@@ -1,62 +1,64 @@
 #Module to load dataset and divide it into train, test and validation sets for X and Y
 
-def Load_dataset(filename):
+import numpy as np
+import matplotlib.pyplot as plt
 
-    import numpy as np
+
+def load_dataset(filename):
 
     try:
```

Comment:
I already commented on this - this try/catch does not give anything. It actually is bad, because it hides error message that would normally be deisplayed. Let the errors be thrown if you are not going to handle the exception.